### DISCUSSION TITLE
Gem Based Install Does Not See _pages Directory

### DISCUSSION THREAD

**Discussion by veaceslavcretu** [mmistakes/minimal-mistakes#2681] [Q&A] [answered]
Hello - 

After installing the theme as as gem, I went ahead and added the directory _pages into the _config.yml as include: - _pages and the files in the directory do not show in the top bar. 

I did try the main https://mmistakes.github.io/minimal-mistakes/docs/navigation/ links and they do not appear as well (data file from the gem folder)

If the theme is downloaded, then the changes work. 

Could you please clarify how to go about? 

Thank you!

**Comment by mmistakes** [comment, marked as answer]
There is no automatic inclusion of navigation links. If you want links in the header/masthead they need to be defined [as documented](https://mmistakes.github.io/minimal-mistakes/docs/navigation/#masthead). If they still don't show up then something could be wrong with your YAML... likely improper indents.

If you can share a public repo so I can look at your code and how things are setup I might be able to spot your error. Otherwise I'm just guessing...
